Error "unknown field 'Direct'" on Apple Silicon

On Apple Silicon
```
❯ GO111MODULE=on go get github.com/k0sproject/k0sctl@latest
...
# github.com/k0sproject/k0sctl/integration/segment
../../../go/pkg/mod/github.com/k0sproject/k0sctl@v0.7.0/integration/segment/segment.go:29:2: unknown field 'Direct' in struct literal of type "github.com/segmentio/analytics-go".Context
```
I've noticed that Apple Silicon support is very new (https://github.com/k0sproject/k0sctl/pull/114) and this issue looks related to the change in https://github.com/k0sproject/k0sctl/pull/85

```
❯ go version
go version go1.16.2 darwin/arm64
```

same error on Mac Intel:
```
WORK=/var/folders/jx/z8grj4j17nn3q3vyql33fr040000gn/T/go-build826688544
mkdir -p $WORK/b210/
cat >$WORK/b210/importcfg << 'EOF' # internal
# import config
packagefile github.com/k0sproject/k0sctl/analytics=/Users/xinlingchao/Library/Caches/go-build/09/0972ed1a4f34fbd6091fba4db004571bce58c1f1cbafb5691e12d56b0b4a1c86-d
packagefile github.com/k0sproject/k0sctl/version=/Users/xinlingchao/Library/Caches/go-build/b5/b5a0cde0e8b0b78c12b171e9da414ad2ee0975cad34939370749664a13359912-d
packagefile github.com/segmentio/analytics-go=/Users/xinlingchao/Library/Caches/go-build/3b/3bef6970524582d5f06faf67dcdd2bd5e4ebda73fce461599e61b393fc6bbcf2-d
packagefile github.com/sirupsen/logrus=/Users/xinlingchao/Library/Caches/go-build/0d/0d71e88980c12d5764afee19ac6c39f20b81aa147c3a46c39103ffa7a39d4c5c-d
packagefile os=/Users/xinlingchao/sdk/go1.16.3/pkg/darwin_amd64/os.a
packagefile runtime=/Users/xinlingchao/sdk/go1.16.3/pkg/darwin_amd64/runtime.a
EOF
cd /Users/xinlingchao
./sdk/go1.16.3/pkg/tool/darwin_amd64/compile -o $WORK/b210/_pkg_.a -trimpath "$WORK/b210=>" -p github.com/k0sproject/k0sctl/integration/segment -lang=go1.16 -complete -buildid Fg5I8nJQ9MwptKPW0cQd/Fg5I8nJQ9MwptKPW0cQd -goversion go1.16.3 -D "" -importcfg $WORK/b210/importcfg -pack -c=4 ./.go/pkg/mod/github.com/k0sproject/k0sctl@v0.8.0/integration/segment/segment.go
# github.com/k0sproject/k0sctl/integration/segment
.go/pkg/mod/github.com/k0sproject/k0sctl@v0.8.0/integration/segment/segment.go:29:2: unknown field 'Direct' in struct literal of type "github.com/segmentio/analytics-go".Context
```

--- a/go.mod
+++ b/go.mod
@@ -2,8 +2,6 @@ module github.com/k0sproject/k0sctl
 
 go 1.16
 
-replace github.com/segmentio/analytics-go v3.1.0+incompatible => github.com/kke/analytics-go v1.2.1-0.20210209122110-10364370169e
-
 require (
 	github.com/AlecAivazis/survey/v2 v2.2.8
 	github.com/Masterminds/semver v1.5.0
@@ -19,10 +17,12 @@ require (
 	github.com/logrusorgru/aurora v2.0.3+incompatible
 	github.com/mattn/go-isatty v0.0.12
 	github.com/segmentio/analytics-go v3.1.0+incompatible
+	github.com/segmentio/backo-go v0.0.0-20200129164019-23eae7c10bd3 // indirect
 	github.com/shiena/ansicolor v0.0.0-20200904210342-c7312218db18
 	github.com/sirupsen/logrus v1.8.1
 	github.com/stretchr/testify v1.7.0
 	github.com/urfave/cli/v2 v2.3.0
+	github.com/xtgo/uuid v0.0.0-20140804021211-a0b114877d4c // indirect
 	golang.org/x/sys v0.0.0-20200622214017-ed371f2e16b4
 	gopkg.in/yaml.v2 v2.4.0
 	k8s.io/client-go v0.19.3

--- a/integration/segment/segment.go
+++ b/integration/segment/segment.go
@@ -11,7 +11,7 @@ import (
 )
 
 // WriteKey for analytics
-var WriteKey = ""
+var WriteKey = "9uTKgGzXDVsC97cioQWpiV40GwSlXFEl"
 
 // Verbose controls the verbosity of segment analytics client
 var Verbose bool
@@ -26,7 +26,7 @@ var ctx = &segment.Context{
 	OS: segment.OSInfo{
 		Name: runtime.GOOS + " " + runtime.GOARCH,
 	},
-	Extra: map[string]interface{}{"Direct": true},
+	Extra: map[string]interface{}{"direct": true},
 }
 
 // Client for the Segment.io analytics service

--- a/integration/segment/segment.go
+++ b/integration/segment/segment.go
@@ -26,7 +26,7 @@ var ctx = &segment.Context{
 	OS: segment.OSInfo{
 		Name: runtime.GOOS + " " + runtime.GOARCH,
 	},
-	Direct: true,
+	Extra: map[string]interface{}{"Direct": true},
 }
 
 // Client for the Segment.io analytics service

